### ISSUE TITLE
aktywacja konta

### ISSUE THREAD

**Issue by filusek** [allegro/allegro-api#9962]
Proszę o weryfikację konta dla lukasz.firlej25@gmail.com

**Comment by filusek** [comment]
Dziękuje.

**Comment by Lukasz-Zurek** [comment]
@mycommercium I have just activated your account.

**Comment by mycommercium** [comment]
Hi,

It is still asking for company details in the sandbox environment and not accepting the fictitious data and documents. Do we have to provide real data for the sandbox? 
![image](https://github.com/user-attachments/assets/9dd4d609-e52f-4323-bc36-0d1fa383e064)
![image](https://github.com/user-attachments/assets/6b5207ec-0570-43df-a25a-1a4489b7c4bd)

**Comment by mycommercium** [comment]
It is still asking for company details in the sandbox environment and not accepting the fictitious data and documents. Do we have to provide real data for the sandbox? Please advise how we can proceed.

![image](https://github.com/user-attachments/assets/03787b71-af05-415e-b7ef-db625c6490f6)
![image](https://github.com/user-attachments/assets/eb8f26c3-fc49-4694-9d66-ee908141fe17)

**Comment by Lukasz-Zurek** [comment]
No, please provide fictitious data. Your account is already active, but if you have some issues and the progress of your account configuration stopped (just like in the screenshot from our guide) - please let me know and share some more details.

**Comment by chr-sandbox** [comment]
Proszę o aktywację konta:
chr-sandbox

Z góry dziękuję

**Comment by mycommercium** [comment]
Hi,

We are asked to verify the bank transfer and confirm the identity in sandbox. We have used fictitious data so we cannot verify it now.  

![image](https://github.com/user-attachments/assets/1c3d9201-c237-4396-a87f-d023ca3038ae)

**Comment by Lukasz-Zurek** [comment]
@mycommercium Please "go to verification". It is a test environment, so there is no real verification, but you have to use our payment simulator.

**Comment by chr-sandbox** [comment]
I can not:

![Zrzut ekranu 2024-09-26 122002](https://github.com/user-attachments/assets/63b7f744-347d-4ac8-b7ef-3b813cb818c7)

![Zrzut ekranu 2024-09-26 122100](https://github.com/user-attachments/assets/103bccbe-9511-4573-a7fa-0417eeda4ece)

**Comment by Lukasz-Zurek** [comment]
@chr-sandbox I have forwarded your case to another department. Please be patient.

**Comment by chr-sandbox** [comment]
Weryfikacja urządzenia się zapętla:
https://allegro.pl.allegrosandbox.pl/moje-allegro/sprzedaz/ustawienia-wyplat....
![Zrzut ekranu 2024-09-26 145948](https://github.com/user-attachments/assets/d001c422-eb04-4b1a-b723-5787d04241a0)

**Comment by Lukasz-Zurek** [comment]
@chr-sandbox Obecnie mogą występować problemy techniczne. Przepraszamy za utrudnienia. 

Jak tylko sytuacja wróci do normy - wrócę z informacją.

**Comment by Lukasz-Zurek** [comment]
@chr-sandbox Spróbuj proszę ponownie [skonfigurować wypłaty](https://allegro.pl.allegrosandbox.pl/moje-allegro/sprzedaz/ustawienia-wyplat) i skorzystać z naszego symulatora płatności, by to zatwierdzić.

**Comment by Lukasz-Zurek** [comment]
@mycommercium Please check it out again and let me know if it is ok.

**Comment by chr-sandbox** [comment]
It works, many thanks
Banking transfer is done (immitation), company's details verification in progress.
Could you verify my company's data
Thank you in advance

**Comment by Lukasz-Zurek** [comment]
@chr-sandbox Ok, I have just verified and the account is active.

**Comment by mycommercium** [comment]
It's verified. Thank you!
